recipes/nbpresent/run_test.sh

```diff
@@ -0,0 +1,11 @@
+
+# On circle-ci, we are building in a container, so CIRCLE_CI is not set
+# So this will have to do.
+if [ `which yum` ]; then
+  yum install -y fontconfig freetype
```

Comment:
phantomjs is really dumb, probably need to build it against those libs, but for the time being, I'll check out `yum_requirements.txt`! It's like christmas!